### DISCUSSION TITLE
People can't change their visibility in organizations more than 50,000 members

### DISCUSSION THREAD

**Discussion by Mehver** [community/community#12534] [Projects and Issues] [answered]
![image](https://user-images.githubusercontent.com/75297777/157197859-9b1a16ba-f470-468b-8bbd-6f1219220bf1.png)

Here I use @EpicGames as an example. In this organization, `https://github.com/orgs/EpicGames/people?query=` , the search function, returns error by timeout. And even if I use crawler to catch down people's name and their page numbers, I can still only get the first 50,000 people's. So after all, users can only stay in private after the get into this org.

**Comment by diamant3** [comment, marked as answer]
You can use this to [fix](https://avestura.dev/blog/change-visibility-in-massive-github-orgs), github support said this is the correct fix for that

**Reply by Mehver** [reply]
It really work! Thanks a lot

**Comment by Chaoses-Ib** [comment]
Here is the legacy issue: https://github.com/isaacs/github/issues/1998
